templates/admin/jecoute/exports.html.twig

```diff
@@ -0,0 +1,14 @@
+{% extends 'SonataAdminBundle:CRUD:base_list_field.html.twig' %}
+
+{% block field %}
+    {% if admin.hasAccess('show') %}
```

Comment:
do we need to do the same check in the controller?

Reply:
yes indeed, and I actually moved this check into the Admin class, so the column is not added if we don't have the access